hack/istio/skupper/install-skupper-demo.sh

```diff
@@ -330,20 +324,20 @@ openshift_install_basic_demo() {
   openshift_login ${CLUSTER1_ISTIO}
 
   infomsg "Installing Istio ..."
-  ${HACK_SCRIPTS_DIR}/istio/install-istio-via-istioctl.sh -c ${CLIENT_EXE}
+  ${HACK_SCRIPTS_DIR}/istio/install-istio-via-istioctl.sh -c ${CLIENT_EXE} -s values.meshConfig.outboundTrafficPolicy.mode=REGISTRY_ONLY
 
   infomsg "Installing Bookinfo demo ..."
   ${HACK_SCRIPTS_DIR}/istio/install-bookinfo-demo.sh -c ${CLIENT_EXE} --traffic-generator --wait-timeout 5m
```

Comment:
```suggestion
  ${HACK_SCRIPTS_DIR}/istio/install-bookinfo-demo.sh -c ${CLIENT_EXE} --traffic-generator --wait-timeout 5m

  infomsg "Updating Bookinfo traffic-generator route ..."
  ${CLIENT_EXE} patch configmap traffic-generator-config -n bookinfo --type merge -p '{"data":{"route":"http://productpage:9080/productpage"}}'
  infomsg "Restarting Bookinfo traffic-generator pod ..."
  ${CLIENT_EXE} delete pod -n bookinfo -l app=kiali-traffic-generator
```

Reply:
the default bookinfo demo install on openshift goes from traffic generator pod through istio ingress, but that is now blocked with REGISTRY_ONLY setting, so just like the minikube install, let's tell it to go directly to the productpage service